design/decisions/why-standardisation/index.qmd

```diff
@@ -0,0 +1,71 @@
+
+### Code snippets in VS Code 
+
+User generated code snippets is a tool that was made available in Visual Studio Code back in 2018.  There is similar functionality in other editors, but VSCode seems to be the one that have support for markdown and quarto.  Code snippets rely on tab completion being enabled, but once that has been set it is very easy to create a set of custom snippets.  For the seedcase team we have (for now) focused on creating snippets that will allow us to use the same markdown/quarto formatting when writing our documentation.  There are snippets for creating tables as well as standardising comments, more information on how to use code snippets can be found here.
+
+<!--TODO link to doc on code snippets once we decide where to place it -->
+
+<!--TODO Check with Richard if pyCharm has similar function for 'text' editing-->
+
+### Joint list of key words for documentation
+
+Quarto allows for the generation of key words that can be used to filter documentation.  It is important for usability of the website that the words used are the same, and spelled the same, so that if a person is looking for how Seedcase handles databases they get everything by clicking on the keyword database, and aren't missing half the documents because they are tagged with the key word databases.  
+
+The following guidelines for selecting key words apply:
+
+* **Form** we prefer using nouns over verbs (eg. documentation over documenting, contribution over contributing)
+* **Plural vs Singular** we prefer singular over plural (eg. database over databases, and container over containers)
+* **More than one word** we prefer to write keywords that consist of more than one word without hyphens (eg. design architecture over design-architecture), the exception is of course where the word is normally spelled with a hyphen (eg. back-end)
+
+The list of key words is included in the code snippet for setting up the title section in the documentation, it is the plan that it will also be accessible through a dedicated page at a later date, but this is still in the design phase.
+
+<!--TODO #207 Add the key words to the title code snippet-->
+
+### Project pages on GitHub
+
+It is debatable whether or not the project pages on GitHub is a tool for standardising workflows.  Here we have included a brief description, further information on how we expect to use it is detailed on this page.
```

Comment:
I agree that I'm not sure whether the project pages is a tool for standardising workflows Should we move this to a separate decision post?